glotaran/__init__.py

```diff
@@ -1,23 +1,9 @@
 """Glotaran package __init__.py"""
 
-from . import io
-from . import parameter
-
 __version__ = "0.3.0"
+# TODO: add git SHA1 information
 
-ParameterGroup = parameter.ParameterGroup
-
-read_parameters_from_csv_file = ParameterGroup.from_csv
-read_parameters_from_yaml = ParameterGroup.from_yaml
-read_parameters_from_yaml_file = ParameterGroup.from_yaml_file
-
-
-from .parse import parser  # noqa: E402
-
-read_model_from_yaml = parser.load_yaml
-read_model_from_yaml_file = parser.load_yaml_file
-
-import pkg_resources  # noqa: E402
+import pkg_resources
```

Comment:
Since pyglotaran is python >= 3.8 we can use the builtin `importlib.metadata.entry_points` and don't need `pkg_resources` anymore.

Reply:
replaced it